src/interactions/decorators.ts

```diff
@@ -369,6 +378,24 @@ export function userContextMenu(name?: string): ApplicationCommandDecorator {
   }
 }
 
+export function messageComponent(customID?: string): MessageComponentDecorator {
+  return function (
+    client: ApplicationCommandClientExt,
+    prop: string,
+    desc: TypedPropertyDescriptor<ComponentInteractionCallback>
+  ) {
+    if (client._decoratedComponents === undefined)
+      client._decoratedComponents = []
+    if (typeof desc.value !== 'function') {
+      throw new Error('@userContextMenu decorator requires a function')
```

Comment:
Shouldn't it say message component not context menu?
```suggestion
      throw new Error('@messageComponent decorator requires a function')
```